index.html

```diff
@@ -0,0 +1,310 @@
+                <!-- skills -->
+                <section class="skill-card">
+                    <!-- icon -->
+                    <div class="skill-card-icon">
+                        <div class="skill-card-icon-container"><img class="skill-card-icon-img" src="img/skill/skills-img.svg" alt=""></div>
+                    </div>
+                    <!-- title and tags -->
+                    <div  class="skill-card-content">
+                        <h3 class="skill-card-title">Skills</h3>
+                        <ul class="skill-card-rows">
+                            <div class="skill-card-tech">
+                                <li class="skill-card-tech-tag codekit"><p class="skill-card-tech-tag-text">Codekit</p></li>
+                                <li class="skill-card-tech-tag github"><p class="skill-card-tech-tag-text">GitHub</p></li>
+                                <li class="skill-card-tech-tag codepen"><p class="skill-card-tech-tag-text">Codepen</p></li>
+                                <li class="skill-card-tech-tag gitlab"><p class="skill-card-tech-tag-text">Gitlab</p></li>
+                            </div>
+                            <div class="skill-card-tech">
+                                <li class="skill-card-tech-tag terminal"><p class="skill-card-tech-tag-text">Terminal</p></li>
+                            </div>
+                        </ul>
+                    </div>
+                </section>
+            </div>
+        </section>
+        
+        <!-- Form Section -->
+        <section id="form">
+            <!-- text -->
+            <div id="form-text">
+                <p id="form-text-item">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            </div>
+            <!-- form -->
+            <form action="https://formspree.io/f/xyyakzgb" method="POST" id="form-content">
+                <div class="form-row" id="first-form-row">
+                    <input type="text" name="FirstName" max-length="30" name="first-name" placeholder="First Name" class="form-row-column" required>
+                    <input type="text" name="SecondName" max-length="30" name="last-name" placeholder="Last Name" class="form-row-column" required>
+                </div>
+                <input type="text" name="FullName" max-length="30" placeholder="Full Name" class="form-row" id="ignoreindesktop" required>
+                <input type="email" name="email" max-length="30" placeholder="Email Address" id="" class="form-row" required>
+                <textarea name="comment" id="" cols="30" rows="10" max-length="500" placeholder="Typing..." class="form-row textareas"></textarea>
+                <div class="button-div">
+                    <button type="submit" class="form-submit-button green-button"><p class="form-submit-button-text">Get in touch </p></button>
+                </div>
+            </form>            
+        </section>
```

Comment:
- [ ] Again I think you are missing the background image for the bottom contact section too please see
![image](https://user-images.githubusercontent.com/97521557/220353978-e41c4a3d-9d6b-4db3-8f8d-6459699d611b.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/220354080-161fd9a6-ddf4-499a-8ae7-cf7b0be7c924.png)